pkg/bootstrap/bootstrap_config.go

```diff
@@ -345,6 +345,10 @@ func writeBootstrapForPlatform(config *meshconfig.ProxyConfig, node string, epoc
 	if cfg == "" {
 		cfg = config.ProxyBootstrapTemplatePath
 	}
+	if _, err := os.Stat(cfg); os.IsNotExist(err) {
+		log.Warnf("Custom template %s not found, fallback to default %s", cfg, DefaultCfgDir)
```

Comment:
If user specifies a custom template falling back to standard template is not the behaviour we want.
It is better than the pods does not start and the user fixes the problem.

Reply:
It's consistent with what we do for the 'configYaml' override - which servers a similar purpose. 

And we don't expect users to specify a custom template - there is no defined way to do it in the installer - we expect users to optionally include an envoy_bootstrap.json in the override configmap.

So user will not actually specify "I want to use the template override" as a values.yaml or annotation - but instead will create a config map with the expected file ( and hopefully have a lot of tests around this - because it's very easy to mess up )